internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -115,6 +116,45 @@ func SetupLogger(seq runtime.Sequence, data interface{}) (runtime.TaskExecutionF
 	}, "setupLogger"
 }
 
+// switchConsole switches the active console to the specified tty.
+func switchConsole(ttyNumber int) error {
```

Comment:
let's please move it to some `internal/pkg/console`, this file is too big already.

Reply:
done